### ISSUE TITLE
Print option

### ISSUE THREAD

**Issue by videosmith** [Cuperino/QPrompt#187]
Occasionally I'm asked to provide hard copy of a script. Is this possible within Qprompt?

**Comment by Cuperino** [comment]
Yes. The print would only have as many details as the prompter’s contents.
If I add the ability to set headings or other metadata, maybe we could have
more adequate formatting to identify who’s speaking and instructions to
play SOT or other things.

Using a dedicated screenwriting app and importing the document would be
ideal IMO, because that way we don’t have to add means to insert and edit
metadata that wouldn’t need to be displayed on the prompter. (Or at the
very least, there wouldn’t be as much need for that.)

Going off topic, for years I’ve wanted to make an app that facilitates
writing scripts in formats suitable for radio, television, and news. I put
it off when I realized my local market wasn’t large enough to justify the
app commercially, but that was before I realized my markets aren’t local.

Since I wouldn’t mind writing this app as open source, I’m considering
repurposing an existing screenwriting app made for film to fit this
purpose. Typically repurposing apps is a bad idea since it can take longer
than writing one from scratch, but the needs here are very similar, so I
think it would actually save time. This is just me thinking out loud, if I
ever give this a go, I’d base the project on the code for Scrite, which is
also written in C++, using Qt and QML.

Coming back to topic:
* What information should the script display, aside from the prompter’s
contents?
* Should it be displayed in a header or footer?
* What other formatting concerns should we take into account?

-- 
Javier O. Cordero Pérez
Full Stack Web Developer
Qt Software Developer

**Comment by videosmith** [comment]
On Tuesday, May 23, 2023 at 12:37:26 PM EDT, Javier O. Cordero Pérez ***@***.***> wrote:  
 
 

Yes. The print would only have as many details as the prompter’s contents.
If I add the ability to set headings or other metadata, maybe we could have
more adequate formatting to identify who’s speaking and instructions to
play SOT or other things.

>> At the present time, I'm only tasked with exporting a basic black text/white background (~12pt. font) Word format document of the script for show runners to follow during the performance although text decorations such as Bold, Italic and Underline would be desirable.

Using a dedicated screenwriting app and importing the document would be
ideal IMO, because that way we don’t have to add means to insert and edit
metadata that wouldn’t need to be displayed on the prompter. (Or at the
very least, there wouldn’t be as much need for that.)''
>> If an html export would honor all attributes of the original document while flipping Black/Automatic text to white while retaining colored text attributes and flipping the white background to Black, would be a great start. (So far, html exports from LibreOffice Writer do not appear to honor all carriage returns and spaces)

Going off topic, for years I’ve wanted to make an app that facilitates
writing scripts in formats suitable for radio, television, and news. I put
it off when I realized my local market wasn’t large enough to justify the
app commercially, but that was before I realized my markets aren’t local.
>> There are applications like "shoflo dot tv", that appear to go in this direction.

Since I wouldn’t mind writing this app as open source, I’m considering
repurposing an existing screenwriting app made for film to fit this
purpose. Typically repurposing apps is a bad idea since it can take longer
than writing one from scratch, but the needs here are very similar, so I
think it would actually save time. This is just me thinking out loud, if I
ever give this a go, I’d base the project on the code for Scrite, which is
also written in C++, using Qt and QML.
Coming back to topic:
* What information should the script display, aside from the prompter’s
contents?
* Should it be displayed in a header or footer?
* What other formatting concerns should we take into account?
>> see first reply please

-- 
Javier O. Cordero Pérez
Full Stack Web Developer
Qt Software Developer

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>


| 
| 
| 
|  |  |

 |

 |
| 
|  | 
All-In-One Live Streaming, Rundown, And Teleprompting Software - Shoflo

An all-in-one solution for live streaming events, building run of shows, and teleprompting.
 |

 |

 |
